cache/cache_test.cc

```diff
@@ -2,10 +2,6 @@
 //  This source code is licensed under both the GPLv2 (found in the
 //  COPYING file in the root directory) and Apache 2.0 License
 //  (found in the LICENSE.Apache file in the root directory).
-//
-// Copyright (c) 2011 The LevelDB Authors. All rights reserved.
```

Comment:
An appropriate copyright line is mandated. (Sorry about mixed signals.) I recommend not touching this for now. (Tech debt: T78128875)

Reply:
@guidotag  You might be confused here.
cache_test.cc was originally from LevelDB, so these lines needs to be kept.
In your new fast_lru_cache_test.cc (abandoned now), these lines are not needed.